### ISSUE TITLE
Potential Use Case: On-boarding for new .NET OSS Developers

### ISSUE THREAD

**Issue by SeanKilleen** [SeanKilleen/AWiderDotNet#3]
A number of sites have attempting to accomplish this already, so I'm thinking we could serve as a springboard for them, as well as filling in any general or .NET-specific gaps we see in the existing process.

Your thoughts?

**Comment by M-Zuber** [comment]
What do you mean by onboarding?
  For some people the concept of `First PR` could mean first in a given project, first for a specific methodology, language etc.

I think it would be worth it to try and keep a list of such resources as reference here in the issue:
Here are some that might be relevant.
- [Article discussing setting up first-timer only friendly issues](https://medium.com/@kentcdodds/first-timers-only-78281ea47455#.l8zc0p1hw)
  - [Site resulting from said article](http://www.firsttimersonly.com/)
- [Github search for first-timer issues - can be edited to include searching on repo language](https://github.com/search?utf8=%E2%9C%93&q=label%3Afirst-timers-only+is%3Aopen&type=Issues&ref=searchresults)
- [Twitter account that tweets about such issues](https://twitter.com/yourfirstpr)
- [As long as it's not December there are a number of links here](http://24pullrequests.com/) - If you want me to include them individually let me know
- [Up for grabs](http://up-for-grabs.net/#/)
- [Looking for Pull Requests](http://www.lookingforpullrequests.com/)

Most of these sites can be searched by language/stack so we can provide links more `.net` focused.
for example: http://up-for-grabs.net/#/tags/.net%2Cc%23%2Cf%23%2Casp.net (there are more tags that can be added - this is just an example)

**Comment by SeanKilleen** [comment]
@M-Zuber great question. By on-boarding, I mean easing people in. This could mean first PR, but it could also mean the initial encouragement to help them seek out and get to the point of submitting their first PR. I mean a general introduction to the culture and a ton of encouragement along the way. 

Those are excellent links, and they're absolutely in line with what I'm thinking. I think we wouldn't try to reinvent those sites, but would make sure that developers had those resources front and center.

When it comes down to it, I think the purpose of this site is to encourage developers, but even moreso to encourage their companies to dive into the ecosystem (having devs who are interested/willing is an important precursor to that).

Thanks for getting involved!

**Comment by M-Zuber** [comment]
> I think we wouldn't try to reinvent those sites, but would make sure that developers had those resources front and center.

Absolutely, but providing links pre filtered to `.net` related items would help - as would making sure those sites have such items in their respective datasets.

> When it comes down to it, I think the purpose of this site is to encourage developers, but even moreso to encourage their companies to dive into the ecosystem (having devs who are interested/willing is an important precursor to that).

I am not going to spam anyone with my history here, but unfortunately I can mainly be helpful only on the dev side. :(
Will do my best all around though.

> Thanks for getting involved!

Thank you for starting it!!
